src/pages/settings/InitialSettingsPage.js

```diff
@@ -52,6 +53,9 @@ const propTypes = {
 
         /** The user's role in the policy */
         role: PropTypes.string,
+
+        /** The current action that is waiting to happen on the policy */
+        pendingAction: PropTypes.oneOf(['add', 'update', 'delete']),
```

Comment:
There are constants for these https://github.com/Expensify/App/blob/3fa24e24a62d4a5c12d17e3386e969a68b922e58/src/CONST.js#L788-L792

Reply:
Good call. Will update.